### ISSUE TITLE
Tenuto's interpretation

### ISSUE THREAD

**Issue by Dima-S-Jr** [musescore/MuseScore#24418]
### Your idea

My idea is this: I propose to implement the following interpretations of _tenuto_:
1. A soft accent (the sound is not taken with an accent, but with a more weighty, operatic, soft and deep touch. "Expressively maintaining" is how Moser interprets the meaning of _tenuto_). In piano playing, _tenuto_ can indeed indicate dynamic note highlighting, but not at the expense of a short accent, but with the help of a deeper, more weighty immersion in the keyboard.
2. Sound overexposure. (This meaning of the term _tenuto_ is noted in some dictionaries, in particular by Garras (1898).

The user should be able to choose all these interpretations at his discretion and apply them accordingly.

### Problem to be solved

At the moment, MuseScore Studio does not interpret the _tenuto_ in any way (whether it exists or not, the sound is the same). There is no soft accent, no slight overexposure of sound. But we must understand that _tenuto_ was introduced into musical terminology for a reason, not for show. And the _tenuto_ notes still have to stand out somehow, sound different. 

Of course, there can be a lot of discussion about this, and everyone can have their own idea of _tenuto_. Yes, first of all, _tenuto_ involves fully maintaining the duration of the note. And based on this interpretation, it would seem that there is no point in interpreting this term in any other way. But let's take note that MuseScore currently interprets all the notes to fit the same style. I definitely can't say that it's _tenuto_ (or _sostenuto_), but _legato_ is probably for sure. And I will say, just in case, that with _legato_, the performer's attention is directed to the connection of neighboring sounds with each other, their flow into each other, and with _tenuto_ it concentrates on each sound individually, on the sound as such. Feinberg writes about this very accurately. He characterizes, however, not _tenuto_, but _sostenuto_, but his explanation perfectly fits _tenuto_. 

There is also a less successful interpretation of _tenuto_, and it lies in the fact that _tenuto_ is a term by its nature purely cautionary, precautionary in nature: it is a warning not to play _staccato_ (Grant). In my opinion, this is exactly the same formulation that MuseScore uses to this day. But what about the existence of _tenuto-staccato_? Agree, there are many questions here. _Tenuto_ in MuseScore is actually quite an interesting thing. By itself, it does not affect the sound in any way, and in some cases it can even spoil in combination with another articulation (see #21700). 

To summarize the above, it would still be necessary to set some special sound for the _tenuto_, i.e. so that the notes from the _tenuto_ stand out. I suggest (at least to begin with) interpreting this as a soft accent, which will give a clear distinction between _tenuto_ notes and _non-tenuto_ notes.

### Checklist

- [X] This request follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting issues
- [X] I have verified that this feature request has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar requests
